deploy/template.yaml

```diff
@@ -771,6 +775,14 @@ Resources:
             Period: !FindInMap [ EnvironmentConfiguration, !Ref AWS::AccountId, tg500ErrorWindow ]
             Stat: Sum
 
+  #AssetsUploadInvoke:
```

Comment:
Why's this commented out? Wouldn't deleting it be preferable?

Reply:
yes - I've removed as it's not ready - will copy back in next week